README.md

```diff
@@ -20,13 +20,14 @@ Review the [Architecture concept](docs/architecture.md) document for more detail
 A few key features: 
 - Support for multiple frameworks. Serve models trained in popular formats such as Caffe*, TensorFlow*, MXNet* and ONNX*.
 - Online deployment of new [model versions](docs/model_version_policy.md).
+- Configuration updates a in runtime
 - Support for AI accelerators including [Intel Movidius Myriad VPUs](https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_VPU.html), 
 [GPU](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_CL_DNN.html) and [HDDL](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_HDDL.html). 
 - The server can be enabled both on [Bare Metal Hosts](docs/host.md) or in
 [Docker containers](docs/docker_container.md).
-- [Kubernetes deployments](deploy). The server can be deployed in a Kubernetes cluster allowing the inference service to scale horizontally and ensure high availability.  
 - [Model reshaping](docs/shape_and_batch_size.md). The server supports reshaping models in runtime. 
 - [Directed Acyclic Graph Scheduler](docs/dag_scheduler.md) Connect multiple models to deploy complex processing solutions and reduce overhead of sending data back and forth.
+- [Custom nodes in DAG pipelines](docs/custom_node_development.md) Model inference can be represented by a custom node implementation attached as a C/C++ library.
```

Comment:
This sentence seems off to me. I think the intent here is to give message that user can implement any
"inference-like" action as a custom node inside DAG and that's why we mention it here beside DAG documentation?

I would rewrite this at least like:
```suggestion
- [Custom nodes in DAG pipelines](docs/custom_node_development.md) Model inference can be implemented by a custom node C/C++ implementation loaded as an external library.
```